src/components/TileGallery.tsx

```diff
@@ -30,7 +30,8 @@ const createStyles = (isHovered: boolean, _isPressing: boolean) =>
       padding: 24,
       gap: 4,
       width: 198,
-      height: 220,
+      height: '100%',
+      minHeight: 220,
```

Comment:
```suggestion
      minHeight: 220,
```
I don't think you need the specified 100% height.